### ISSUE TITLE
Feature Request: Hide/unhide inbox tabs

### ISSUE THREAD

**Issue by ymyke** [leggett/simplify#684]
I'd love to be able to hide/unhide the inbox tabs, specifically "social" and "promotions" in my case (and still have mails sorted in those buckets behind the scenes). Reason: 1) I don't even want to see that there's new mail in those buckets and 2) I use keystrokes to cycle between "primary" and "updates" but always have to pass through the others (b/c I only go in one direction) and would like to do that quicker.

![image](https://user-images.githubusercontent.com/8012482/169024235-244c884c-3c2a-4614-9729-180445ec8c2d.png)

**Comment by leggett** [comment]
Hi there. Gmail still goes to the Social tab even if it is hidden or reordered with CSS. So, I can't really do this unless I entirely replace the `~` shortcut which I'm not super excited to do.

I do wonder if you'll love [bundles](https://www.fastcompany.com/90698042/simplify-gmail-inbox-bundles) when they are fully ready. Instead of tabs, you get a single line item in your inbox at the place of the last message. Here is part of my current inbox with the [Simplify v3 beta](https://beta.simpl.fyi):

<img width="474" alt="image" src="https://user-images.githubusercontent.com/8763662/169106800-4ed764e7-8d9a-495f-8b35-80257c8755f1.png">

The cool thing is you can do this for Gmail's categories AND for your own labels (if you label things with filters). And the cherry on top (which really addresses your request) is I hope to offer an option to schedule when these bundles are delivered / show up.

TL;DR; I'm going to close this issue but I hope to address the need with bundles.

**Comment by ymyke** [comment]
Thanks, bundles do look interesting indeed.
